org.uqbar.project.wollok.lib/src/wollok/lib.wlk

```diff
@@ -183,6 +183,7 @@ object game {
 	method whenKeyPressedSay(key, function) native
 	method whenCollideDo(element, action) native
 	method getObjectsIn(position) native
+	method colliders(visual) native
```

Comment:
Can we add a doc to it ? Not sure what it does :)
Returns a set of objects that currently collides with the given visual object ?